lib/manager/maven/readme.md

```diff
@@ -0,0 +1,138 @@
+## Overview
+
+#### Name of package manager
+
+Maven
+
+---
+
+#### What language does this support?
+
+Java
+
+---
+
+#### Does that language have other (competing?) package managers?
+
+Gradle 
+
+## Package File Detection
+
+#### What type of package files and names does it use?
+
+"pomfile" usually named pom.xml. It is possible to override default pom.xml with any filename
```

Comment:
If that file name override defined anywhere in the git repo that we could detect?

Reply:
we can detect it based on content if needed. 
Take all xml files and check their headers
pom should have something like that 
`<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">`

Reply:
How many "false hits" do you think we'd have? i.e. are xml files pretty common in Java projects?